src/BaGet.GCP/Services/GoogleBucketStorageService.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.IO;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using BaGet.Core.Services;
+using BaGet.GCP.Configuration;
+using Google;
+using Google.Cloud.Storage.V1;
+using Microsoft.Extensions.Options;
+
+namespace BaGet.GCP.Services
+{
+    public class GoogleBucketStorageService : IStorageService
+    {
+        private readonly string _bucketName;
+
+        public GoogleBucketStorageService(IOptionsSnapshot<GoogleBucketStorageOptions> options)
+        {
+            if (options == null)
+                throw new ArgumentNullException(nameof(options));
+
+            _bucketName = options.Value.BucketName;
+        }
+
+        public async Task<Stream> GetAsync(string path, CancellationToken cancellationToken = default)
+        {
+            var storage = await StorageClient.CreateAsync();
```

Comment:
`StorageClient` is disposable, please wrap all `StorageClient.CreateAsync()` with `using`s.